data/gegno/gegno.txt

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2022 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+phrase "gegno vi"
+	word
+		""
+		""
+		""
+		""
```

Comment:
```suggestion
		"" 4
```
this is the same thing but in less lines

Reply:
This works this way? Are you sure? I don't want 4 spaces, but I want a larger chance of there being a space. If it does, cool.

Reply:
Just looked at the wiki - I didn't realize phrases have weights now.

Reply:
yeah that's what this does, its already used for the Hai in fact, in their phrases